packages/paste-core/components/breadcrumb/__tests__/breadcrumb.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {axe} from 'jest-axe';
+import {matchers} from 'jest-emotion';
+import {render, screen} from '@testing-library/react';
+import {Breadcrumb, BreadcrumbItem} from '../src';
+
+expect.extend(matchers);
+
+describe('Breadcrumb', () => {
+  it('it should render a nav with correct aria-label', () => {
+    const {getByRole} = render(
```

Comment:
nit: are you able to move this render into the describe() { block so that we can use it multiple times? Haven't tried it so not sure, but could speedup the test.

Reply:
Good suggestion. That sped the tests up from about 7s to 3.5s.